source/mir/ndslice/slice.d

```diff
@@ -316,14 +317,17 @@ pure nothrow @nogc unittest
     }
 
     alias S = Slice!(3, MyIota);
-    auto slice = MyIota().sliced(20, 10);
-
     import std.range.primitives;
     static assert(hasLength!S);
     static assert(hasSlicing!S);
     static assert(isForwardRange!S);
     static assert(isBidirectionalRange!S);
     static assert(isRandomAccessRange!S);
+
+    auto slice = MyIota().sliced(20, 10);
+    assert(slice[0, 0] == 0);
+    auto sCopy = slice.save;
+    assert(sCopy[0, 0] == 0);
```

Comment:
Hmm the problem is that the `struct` counts to the coverage as miss :/
Other ideas?